code/modules/food_and_drinks/food/snacks_soup.dm

```diff
@@ -225,3 +225,11 @@
 	bonus_reagents = list("nutriment" = 4, "vitamin" = 6)
 	tastes = list("beet" = 1)
 	foodtype = VEGETABLES
+
+/obj/item/reagent_containers/food/snacks/soup/monkey
+	name = "sopa de macaco"
+	desc = "Uma delicia"
+	icon_state = "sopademacaco"
+	bonus_reagents = list("nutriment" = 10, "vitamin" = 30)
```

Comment:
why is this ten times higher than most food types

Reply:
Because you’re eating an entire body?

Reply:
vitamins are for healthy food not stuffing a monkey corpse into a bowl

Reply:
Well an entire body probably has a few vitamins

Reply:
you need a full monkey to make it

Reply:
You need a full monkey to make a super bite burger and a whole bunch of other stuff and it still has half the vitamins than this

Don't pretend you even tried to balance this

Reply:
ok it has more vitamins because it's delicia